ci/pipelines/concourse.yml

```diff
@@ -782,20 +798,174 @@ jobs:
       vars_files:
       - prod/wings/vars.yml
 
-- name: push-docs
+- name: shipit
   public: true
+  serial_groups: [version]
+  plan:
+  - get: concourse
+    passed:
+    - build-rc
+    - k8s-topgun
+    - bosh-smoke
+    - bosh-topgun
+  - get: unit-image
+    passed:
+    - build-rc
+    - k8s-topgun
+    - bosh-smoke
+    - bosh-topgun
+  - get: final-version
+    resource: version
+    params: {bump: final}
+    passed:
+    - build-rc
+    - k8s-topgun
+    - bosh-smoke
+    - bosh-topgun
+  - get: linux-rc
+    passed: [build-rc]
+  - get: windows-rc
+    passed: [build-rc]
+  - get: darwin-rc
+    passed: [build-rc]
+  - get: concourse-rc-image
+    passed: [k8s-topgun]
+  - get: charts
```

Comment:
Hey, could you clarify why there's the need of having a `get` on `charts`? Is it just to pin down which version was used at the time?

Thanks!

Reply:
Yeah, the idea is for `shipit` to be the gateway and mark everything that we want to ship so that downstream publishing jobs know exactly what version to use. We don't currently do anything with `charts` but I figured we might at least want to know the sha that made it through when it comes time to manually publish.